spec.bs

```diff
@@ -6419,11 +6443,23 @@ The following algorithm will be added to the [[FETCH#fetching]] section:
       that is, scripts making the {{WindowOrWorkerGlobalScope/fetch()}} request aren't able to load
       the header value.
     1. [=list/For each=] |bid| of |additionalBids|:
-      1. Let |nonceAndAdditionalBid| be the result of [=strictly splitting=] |bid| on U+003A (:).
-      1. If |nonceAndAdditionalBid|'s [=list/size=] is not 2, then [=iteration/continue=].
-      1. Let |nonce| be |nonceAndAdditionalBid|[0].
-      1. If |nonce|'s [=string/length=] is not 36, then [=iteration/continue=].
-      1. Set |storedAdditionalBidsHeaders|[|nonce|] to |nonceAndAdditionalBid|[1].
+      1. Let |parts| be the result of [=strictly splitting=] |bid| on U+003A (:).
+      1. Let |bidWithMetadata| be a new [=signed additional bid with metadata=].
+      1. If |parts|'s [=list/size=] is 3:
+        1. Let |auctionNonce| be |parts|[0].
+        1. If |auctionNonce|'s [=string/length=] is not 36, then [=iteration/continue=].
+        1. Let |sellerNonce| be |parts|[1].
+        1. If |sellerNonce|'s [=string/length=] is not 36, then [=iteration/continue=].
+        1. Let |bidWithMetadata|'s [=signed additional bid with metadata/seller nonce=] be
+          |sellerNonce|.
+        1. Let |bidWithMetadata|'s [=signed additional bid with metadata/signed additional bid=] be
+          |parts|[2].
+        1. Set |storedAdditionalBidsHeaders|[|auctionNonce|] to |bidWithMetadata|.
+      1. Otherwise, if |parts|'s [=list/size=] is 2: 
+        1. Let |auctionNonce| be |parts|[0].

```

Comment:
Missing:
1. If |auctionNonce|'s [=string/length=] is not 36, then [=iteration/continue=].

Reply:
Done.